streams/pe.m3u

```diff
@@ -75,8 +75,8 @@ https://s1.tvdatta.com:3804/live/diasporatvlive.m3u8
 https://v4.tustreaming.cl/s1tv/playlist.m3u8
 #EXTINF:-1 tvg-id="DobleCTelevision.pe",Doble C Televisión (Mazamari) [Not 24/7]
 https://7.innovatestream.pe:19360/cctvchupaca/cctvchupaca.m3u8
-#EXTINF:-1 tvg-id="ExitosaTV.pe",Exitosa TV (720p)
-https://cu.onliv3.com/livevd1/user2.m3u8
+#EXTINF:-1 tvg-id="Exitosa.pe",Exitosa (720p)
```

Comment:
The ID must be the same as in [iptv-org/database](https://github.com/iptv-org/database).

```suggestion
#EXTINF:-1 tvg-id="ExitosaTV.pe",Exitosa (720p)
```

https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#stream-description-scheme